exporter/kafkaexporter/authentication.go

```diff
@@ -68,6 +87,32 @@ func configurePlaintext(config PlainTextConfig, saramaConfig *sarama.Config) {
 	saramaConfig.Net.SASL.Password = config.Password
 }
 
+func configureSASL(config SASLConfig, saramaConfig *sarama.Config) error {
+
+	if config.Username == "" || config.Password == "" {
+		return fmt.Errorf("username and password cannot be empty")
```

Comment:
```suggestion
		return fmt.Errorf("both username and password have to be provided")
```
I personally prefer one check per property, so that we can tell the user which specific property we are complaining about.

Reply:
I totally agree, changed to one check per property.